x/mint/module_simulation.go

```diff
@@ -19,13 +17,13 @@ func (AppModule) GenerateGenesisState(simState *module.SimulationState) {
 }
 
 // ProposalContents doesn't return any content functions for governance proposals.
-func (AppModule) ProposalContents(simState module.SimulationState) []simtypes.WeightedProposalContent {
+func (AppModule) ProposalContents(_ module.SimulationState) []simtypes.WeightedProposalContent { //nolint:staticcheck
```

Comment:
I believe `ProposalContents` and `ProposalMsgs` could be removed if no proposal is being tested